shared/readLFP.m

```diff
@@ -55,13 +55,18 @@
     fprintf('*** Loading precomputed LFP data ***\n');
     fprintf('************************************\n\n');
     load(fname_out,'LFP');
+    return
```

Comment:
Very nice! didn't spot that one, will do it in the other functions as well every time a new one is committed.